src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -228,12 +228,15 @@ private void prepareStatements() {
         = session.prepare("DELETE FROM repair_run_by_cluster WHERE id = ? and cluster_name = ?");
     deleteRepairRunByUnitPrepStmt = session.prepare("DELETE FROM repair_run_by_unit "
         + "WHERE id = ? and repair_unit_id= ?");
-    insertRepairUnitPrepStmt =
-        session.prepare(
+    insertRepairUnitPrepStmt = session
+        .prepare(
             "INSERT INTO repair_unit_v1(id, cluster_name, keyspace_name, column_families, "
                 + "incremental_repair, nodes, datacenters, blacklisted_tables, repair_thread_count) "
-                + "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)");
-    getRepairUnitPrepStmt = session.prepare("SELECT * FROM repair_unit_v1 WHERE id = ?");
+                + "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)")
+        .setConsistencyLevel(ConsistencyLevel.QUORUM);
+    getRepairUnitPrepStmt = session
+        .prepare("SELECT * FROM repair_unit_v1 WHERE id = ?")
+        .setConsistencyLevel(ConsistencyLevel.QUORUM);
```

Comment:
Thanks for the push back. This needs a bit of thought.

`LOCAL_QUORUM` makes sense when an operation is _bound_ to a datacenter.

Because this is more about an internal construct and database integrity, ie see `RepairUnitService.getNewOrExistingRepairUnit(..)` and `Optional<RepairUnit> IStorage.getRepairUnit(RepairUnit.Builder)`, i presumed that the stricter QUORUM would be required, in a similar manner to clusters, schedules and repair runs.

This integrity of the database, and because reapers instances across regions require consistency (or to be to get the data) for clusters, schedules and repair runs, kinda leaves us stuck with QUORUM.
(Segments get to use LOCAL_QUORUM because their activity is within the leader election mechanism.)

I see the choice here as between:
 - implement _eventual consistency_, allowing retries to find repair units (also increasing consistency levels as required),
 - going strict (QUORUM), and if need be caching repair units inside reaper (repair units are immutable, and the table small, so there's no reason to ever fetch a unit twice)

@adejanovski ?